### PR TITLE
Adding bomb indicator. Adding 0 strength wind. Adding Sound Effects.

### PR DESCRIPTION
**Pull request by riverole**
- Added the bomb queue
- Made the 0 wind.
- Some renaming of variables, etc... to follow conventions.
- Adding comments from previous PR.
- Adding sound effects and background music.

BombIndicator

### CODE REVIEW

File: Assets/Scripts/BombSelector.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        PopulateNextBomb(2);
+        return nextBomb;
+    }
+
+    private GameObject ChooseNextBomb()
+    {
+        return bombPrefabs[Random.Range(0, bombPrefabs.Length)];
+    }
+
+    private void PopulateNextBomb(int index)
```
Comment (billykatz) [inline Assets/Scripts/BombSelector.cs]: I dont think we use the `index` passed in

File: Assets/Scripts/BombSelector.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        for (int index = 0; index < maxBombs; index++)
+        {
+            PopulateNextBomb(index);
+        }
+    }
+
+    public GameObject GetNextBomb()
+    {
+        GameObject nextBomb = DequeueBomb();
+        PopulateNextBomb(2);
```
Comment (billykatz) [inline Assets/Scripts/BombSelector.cs]: I think we should rename this function.  It sounds like a getter, but we also add a new bomb to our queue.  Not a big deal- but maybe worth a comment

**Review by billykatz — APPROVED** [review]
Awesome job on this Marc.  I left a small comment but I think we could push this

File: Assets/Scripts/BombSelector.cs (hunk 1 above)
Comment (riverole) [reply Assets/Scripts/BombSelector.cs]: Right, it was leftover from before.

File: Assets/Scripts/BombSelector.cs (hunk 2 above)
Comment (riverole) [reply Assets/Scripts/BombSelector.cs]: Renamed!

**Pull request merged**